framework/CodeInterfaces/PHISICS/MaterialParser.py

```diff
@@ -0,0 +1,157 @@
+"""
```

Comment:
General Comment on this file.

- Fix method names (must be all camelBack)
- Fix docstrings. Check the standards at  https://github.com/idaholab/raven/wiki/Developer_Information)
- Fix camelBack 
- ```__init__``` at the begin of the class